AppKit/CPView.j

```diff
@@ -3392,6 +3431,153 @@ setBoundsOrigin:
 
 @end
 
+@implementation CPView (TrackingAreaAdditions)
+
+- (void)addTrackingArea:(CPTrackingArea)trackingArea
+{
+    // Consistency check
+    if (!trackingArea || [_trackingAreas containsObjectIdenticalTo:trackingArea])
+        return;
+    
+    if ([trackingArea view])
+        [CPException raise:CPInternalInconsistencyException reason:"Tracking area has already been added to another view."];
+
+    [_trackingAreas addObject:trackingArea];
+    [trackingArea setView:self];
+    
+    if (_window)
+        [_window _addTrackingArea:trackingArea];
+  
+    [trackingArea _updateActualRect];
+}
+
+- (void)removeTrackingArea:(CPTrackingArea)trackingArea
+{
+    // Consistency check
+    if (!trackingArea)
+        return;
+    
+    if (![_trackingAreas containsObject:trackingArea])
+        [CPException raise:CPInternalInconsistencyException reason:"Trying to remove unreferenced trackingArea"];
+
+    [self _removeTrackingArea:trackingArea];
+}
+
+// Invoked automatically when the view’s geometry changes such that its tracking areas need to be recalculated.
+
+- (void)updateTrackingAreas
+{
+    // You should override this method to remove out of date tracking areas and add recomputed tracking areas;
```

Comment:
These comments should be in the header of the method (with a /*! comment) so they will get picked up by the documentation generator.